src/qmflows/packages/orca.py

```diff
@@ -111,12 +110,12 @@ def format_hessian(dim: int, hess: List[List[float]]) -> str:
             hess_str += '\n\n$end\n'
 
             # Store the hessian in the plams_dir
-            hess_path = get_tmpfile_name()
+            hess_path = get_tmpfile_name("ORCA_hessian_")
             with open(hess_path, "w") as hess_file:
                 hess_file.write(hess_str)
 
             settings.specific.orca.geom.InHess = "read"
-            settings.specific.orca.geom.InHessName = '"' + hess_path + '"'
+            settings.specific.orca.geom.InHessName = f"{hess_path.as_posix()}"
```

Comment:
Does it have to be double quotation marks (``"``) here?
If not you could also just call ``repr()``.

Reply:
thanks! that is a bug that I just introduced!